crates/biome_css_analyze/src/utils.rs

```diff
@@ -109,3 +111,41 @@ pub fn is_function_keyword(value: &str) -> bool {
 pub fn is_custom_function(value: &str) -> bool {
     value.starts_with("--")
 }
+
+// Returns the vendor prefix extracted from an input string.
+pub fn vender_prefix(prop: &str) -> String {
+    let re = Regex::new(r"^(-\w+-)").unwrap();
+    let vendor_prefix = re
+        .captures(prop)
+        .map(|caps| caps[0].to_string())
+        .unwrap_or_default();
+
+    vendor_prefix
+}
+
+pub fn is_pseudo_elements(prop: &str) -> bool {
+    LEVEL_ONE_AND_TWO_PSEUDO_ELEMENTS.contains(&prop)
+        || VENDOR_SPECIFIC_PSEUDO_ELEMENTS.contains(&prop)
+        || SHADOW_TREE_PSEUDO_ELEMENTS.contains(&prop)
+        || [
+            "backdrop",
+            "content",
+            "cue",
+            "file-selector-button",
+            "grammar-error",
+            "highlight",
+            "marker",
+            "placeholder",
+            "selection",
+            "shadow",
+            "slotted",
+            "spelling-error",
+            "target-text",
+            "view-transition",
+            "view-transition-group",
+            "view-transition-image-pair",
+            "view-transition-new",
+            "view-transition-old",
```

Comment:
These values should be defined in `keyword.rs`

Reply:
Thank you!
I defined it in `keyword.rs` with the following commit.
a9b4e0d15c33b61fef363e0fd69e849711714369